build.sbt

```diff
@@ -27,5 +27,7 @@ libraryDependencies ++= Seq(
 versionIntroduced := Map(
   "3.0.0-M1" -> "1.0.1",
   "3.0.0-M2" -> "1.0.1",
-  "3.0.0-M3" -> "1.0.1"
+  "3.0.0-M3" -> "1.0.1",
+  "3.0.0-RC1" -> "1.1.0",
+  "3.0.0-RC2" -> "1.1.1",
```

Comment:
This might need to be revisited on master.  Or it might not, since I don't think we're doing MiMa on Scala 3 anyway.